README.md

```diff
@@ -22,184 +22,7 @@ Red Hat modifications against the upstream Che include:
-* `CHE_IMAGE_TAG`: `che-server` Docker image tag that will be used for deployment (default is `nightly-fabric8`)
-* `CHE_LOG_LEVEL`: Log level of che-server (default is `DEBUG`)
-* `CHE_DEBUGGING_ENABLED`: If set to `true` the script will create the OpenShift service to debug che-server (default is `true`)
-* `CHE_KEYCLOAK_DISABLED`: If this is set to true Keycloack authentication will be disabled (default is `true` for ocp and minishift, `false` for osio)
-* `OPENSHIFT_NAMESPACE_URL`: The Che application hostname (default is unset for ocp, `${CHE_OPENSHIFT_PROJECT}.$(minishift ip).nip.io` for minishift, `${CHE_OPENSHIFT_PROJECT}.8a09.starter-us-east-2.openshiftapps.com` for osio)
-
-#### ocp and minishift only options
-
-* `OPENSHIFT_USERNAME`: username to login on the OpenShift cluster. Ignored if `OPENSHIFT_TOKEN` is set (default is `developer`)
-* `OPENSHIFT_PASSWORD`: password to login on the OpenShift cluster. Ignored if `OPENSHIFT_TOKEN` is set (default is `developer`)
-
-__Warning__: If you are deploying the RH distribution build, ensure that you created / tagged the Che docker images 
-*in the Minishift docker environment* (see [previous section](#in-the-minishift-docker-environment)).
-If you want to build and deploy the RH distribution to Minishift in one go, you can use the
-[*all-in-one scripts*](#all-in-one-scripts-for-minishift).
-
-### Delete all resources and clean up in Minishift
-
-```bash
-dev-scripts/openshift_deploy.sh --command cleanup
-```
-
-### Roll-update the current Minishift deployment with the up-to-date docker image
-
-```bash
-dev-scripts/openshift_deploy.sh --command rollupdate
-```
-
-__Warning__: If you are deploying the RH distribution build, ensure that you created / tagged the Che docker images
-[*in the Minishift docker environment*](#in-the-minishift-docker-environment).
-If you want to build and deploy the RH distribution to Minishift in one go, you can use the
-[*all-in-one scripts*](#all-in-one-scripts-for-minishift)
-
-### All-in-one scripts for Minishift
-
-Instead of running a script for each step in the process of building / deploying to Minisift,
-the following all-in-one scripts are also available, and can take the [same arguments as
-the build scripts](#build-scripts-parameters)
-
-##### For building the RedHat Che distribution
-
-- `dev-scripts/minishift_build_fabric8_and_deploy.sh [PARAMETERS]`:
-    - changes the current docker environment to use
-the minishift docker daemon,
-    - runs `dev-scripts/build_fabric8.sh [PARAMETERS]`
-    - runs `dev-scripts/openshift_deploy.sh --command cleanup`
-    - runs `dev-scripts/openshift_deploy.sh`
-
-- `dev-scripts/minishift_build_fabric8_and_rollupdate.sh [PARAMETERS]`:
-    - changes the current docker environment to use
-the minishift docker daemon,
-    - runs `dev-scripts/build_fabric8.sh [PARAMETERS]`
-    - runs `dev-scripts/openshift_deploy.sh --command rollupdate`
```

Comment:
We have other dev instructions in this doc below: `Maven build details`, `Enabling / Disabling the Dashboard`, `Running selenium tests`, `VM options`, `Env variables`.
Maybe we don't need them anymore or could move content to the dev guide doc?

Reply:
Moved or removed most of these, thanks.